pt-BR/mail/chrome/messenger-newsblog/feed-subscriptions.dtd

```diff
@@ -1,5 +1,5 @@
 <!-- Subscription Dialog -->
-<!ENTITY newsBlogSubscriptions.label "RSS Subscriptions">
+<!ENTITY newsBlogSubscriptions.label "Assinaturas RSS">
```

Comment:
I agree, however "Assinatura/s" is a word already reserved to be used later, when composing an e-mail message (after completing the e-mail body structure, when we shall insert automatically our name). Therefore, we must distinguish one from another.